### DISCUSSION TITLE
Increment carefully?

### DISCUSSION THREAD

**Discussion by djmcgreal-cc** [renovatebot/renovate#19194] [Request Help] [answered]
Hi, in EKS I can only update a minor version at a time. E.g. I'm on 1.21 and need to get to 1.24 via 1.22, 1.23.
Can I configure renovate to support this sequence?

Thanks.

PS you might be wondering how I'm tracking EKS versions. I have a job that's doing this:
```sh
eksctl version -o json | jq -r '.EKSServerSupportedVersions[-1]' > $VERSION_FILE
gitlab generic-package upload --project-id $CI_PROJECT_ID --package-name "EKS-D" --package-version "$(cat $VERSION_FILE)" --file-name $VERSION_FILE --path $VERSION_FILE
```

And a line like this in my .tf (with a supporting `regexManager`):
```terraform
  cluster_version = "1.21" #renovate: datasource=gitlab-packages dependency=aiinfra/renovate:EKS-D versioning=semver-coerced registry=https://gitlab.com
```

**Comment by viceice** [comment, marked as answer]
No. see
- https://github.com/renovatebot/renovate/issues/6576

**Reply by djmcgreal-cc** [reply]
Thanks
